permit.md

```diff
@@ -30,6 +32,8 @@ select * from vps_vessel where (date_issued<>date_canceled or date_canceled is n
 * Similarly, you may  want to exclude rows where start_date >=end_date or start_date>=date_expired 
  
 *  BUS_OWN and VPS_OWNER are used to aggregate multi-vessel firms together for Regulatory Flexibility Act Analysis. 
+*  NOTE,*the BUS_OWN table is not the main view for business onwer information. THat is found here: Client.bus_own@garfo_nefsc. For more information on this  system see Client.md
```

Comment:
*  NOTE, the BUS_OWN table is not the main view for business owner information. That is found here: Client.bus_own@garfo_nefsc. For more information on this  system see [Client.md](Client.md)